src/Components/WebAssembly/WebAssembly/test/Hosting/TestWebAssemblyJSRuntime.cs

```diff
@@ -1,25 +1,31 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.JSInterop.WebAssembly;
-using Moq;
+using System;
+using Microsoft.AspNetCore.Components.WebAssembly.Services;
 
 namespace Microsoft.AspNetCore.Components.WebAssembly.Hosting
 {
-    public class TestWebAssemblyJSRuntime
+    internal class TestWebAssemblyJSRuntimeInvoker : WebAssemblyJSRuntimeInvoker
     {
-        public static WebAssemblyJSRuntime Create(string environment = "Production")
-        {
-            var jsRuntime = new Mock<WebAssemblyJSRuntime>();
-            jsRuntime.Setup(j => j.InvokeUnmarshalled<object, object, object, string>("Blazor._internal.getApplicationEnvironment", null, null, null))
-                .Returns(environment)
-                .Verifiable();
+        private readonly string _environment;
 
-            jsRuntime.Setup(j => j.InvokeUnmarshalled<string, object, object, byte[]>("Blazor._internal.getConfig", It.IsAny<string>(), null, null))
-                .Returns((byte[])null)
-                .Verifiable();
+        public TestWebAssemblyJSRuntimeInvoker(string environment = "Production")
```

Comment:
Is there any particular reason the default is set to `Production` in the implementation that is designed to run in tests?

Reply:
In the absence of any specific value, the client code assumes `Production`. This mirrors what ASP.NET Core's hosting does too. Responding with `Production` would most closely approximate the actual code.